lib/grammers-client/src/client/net.rs

```diff
@@ -52,6 +67,7 @@ pub(crate) async fn connect_sender(
     let addr: ServerAddr = if let Some(ref sa) = config.params.server_addr {
         sa.clone()
     } else {
+        #[cfg(all(not(all(target_arch = "wasm32", target_os = "unknown")),))]
```

Comment:
Outer `all` seems redundant, unless we're missing a `feature = "proxy"` as we do in sender's `net/mod`.

Reply:
You're right. I missed that when moving both code blocks into a single one. I'll fix it with a force push.